### ISSUE TITLE
MultiSelect selected tags are not announced by Windows Narrator

### ISSUE THREAD

**Issue by Juveniel** [telerik/kendo-ui-core#5589]
### Bug report

MultiSelect selected tags are not announced by Windows Narrator with Edge Chromium.

### Reproduction of the problem

[https://demos.telerik.com/kendo-ui/multiselect/index](https://demos.telerik.com/kendo-ui/multiselect/index)

### Current behavior

MultiSelect selected tags are not announced

### Expected/desired behavior

MultiSelect selected tags should be announced. This will also resolve issues when tags are deselected, as the selection will be announced correctly.

### Environment

* **Kendo UI version:** 2020.1.114
* **Browser:** [ Edge Chromium ]

**Comment by veselints** [comment]
Related to:
https://github.com/telerik/kendo-ui-core/issues/6214

Fixed with:
https://github.com/telerik/kendo/pull/11989

**Comment by veselints** [comment]
Possibly implemented with:
https://github.com/telerik/kendo/issues/12164

If not, leave it open.

**Comment by Dimitar-Goshev** [comment]
The reported behavior is still observed in Edge Chromium and Narrator.
